imi/src/main/java/org/motechproject/nms/imi/service/impl/CdrFileServiceImpl.java

```diff
@@ -1109,6 +1113,8 @@ public void cdrProcessPhase5(MotechEvent event) {
             return;
         }
 
+        eventRelay.broadcastEventMessage(new MotechEvent(CSR_VERIFIER_CACHE_EVICT_MESSAGE));
```

Comment:
should we move this to the beginning of the method (before file copy) so that the cache evict message has guarateed time to run before processCsrs kicks off? 

Not that we should have implicit timing dependencies in code, but just a thought.